stories/imdb/stories/greyVest.js

```diff
@@ -310,9 +310,12 @@ export default () => (
             <Tab value="analytics" label="Analytics">
               <TermsStatsTable
                 tree={tree}
+                criteria={['root', 'criteria']}
+                criteriaField="genres"
                 path={['root', 'genreScores']}
                 tableAttrs={{ className: 'gv-table' }}
                 sizeOptions={[10, 25, 50]}
+                getValue="key"
```

Comment:
What's this? I don't see a `getValue` key on TermsStatsTable - should it be `criteriaGetValue`? 

(Pinging @Akrion since I think this comes from his PR)